src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
My gut tells me we should completely eradicate ```originalEvent``` rather than leaving it part of the structure, i.e.,

```JS
if ( "originalEvent" in e ) {
  delete e.originalEvent;
}
```

because that would be consistent with the way a native event would look. Nevertheless, I don't have enough experience to be able to decide which way of getting rid of it is best. Either way, it looks like something reasonable (like null or undefined) and falsy works.